From 8d5c6986d052ea34fdb3fe0a1affcc7a26f56222 Mon Sep 17 00:00:00 2001
From: Viktor Hofer <viktor.hofer@microsoft.com>
Date: Tue, 7 May 2024 11:11:47 +0200
Subject: [PATCH] Update InternalServerError.cs

The license header was missing which caused unified-build to fail.

>     D:\a\_work\1\vmr\src\aspnetcore\src\Http\Http.Results\src\InternalServerError.cs(1,1): error IDE0073: A source file is missing a required header. (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0073) [D:\a\_work\1\vmr\src\aspnetcore\src\Http\Http.Results\src\Microsoft.AspNetCore.Http.Results.csproj]
---
 src/Http/Http.Results/src/InternalServerError.cs | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/Http/Http.Results/src/InternalServerError.cs b/src/Http/Http.Results/src/InternalServerError.cs
index d9e0aa1a1e08..d6fb8a13b93a 100644
--- a/src/Http/Http.Results/src/InternalServerError.cs
+++ b/src/Http/Http.Results/src/InternalServerError.cs
@@ -1,3 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
 using System.Reflection;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http.Metadata;
@@ -50,4 +53,4 @@ static void IEndpointMetadataProvider.PopulateMetadata(MethodInfo method, Endpoi
 
         builder.Metadata.Add(new ProducesResponseTypeMetadata(StatusCodes.Status500InternalServerError, typeof(void)));
     }
-}
\ No newline at end of file
+}
